packages/react-components/src/components/VideoGallery/ParticipantVideoTileOverlay.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(reaction) */
+import React, { useCallback, useEffect, useState } from 'react';
+/* @conditional-compile-remove(reaction) */
+import { Reaction, ReactionResources } from '../../types';
+/* @conditional-compile-remove(reaction) */
+import { getEmojiResource } from './utils/videoGalleryLayoutUtils';
+/* @conditional-compile-remove(reaction) */
+import { Stack, mergeStyles } from '@fluentui/react';
+/* @conditional-compile-remove(reaction) */
+import { reactionRenderingStyle, videoContainerStyles } from '../styles/VideoTile.styles';
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * Reaction overlay component for Grid
+ *
+ * Can be used with {@link MeetingReactionOverlay}.
+ *
+ * @internal
+ */
+export const ParticipantVideoTileOverlay = React.memo(
+  (props: { reaction?: Reaction; reactionResources: ReactionResources; emojiSize?: number }) => {
+    const { reaction, reactionResources, emojiSize = 44 } = props;
+    const [isValidImageSource, setIsValidImageSource] = useState<boolean>(false);
+
+    const backgroundImageUrl =
+      reaction !== undefined && reactionResources !== undefined
+        ? getEmojiResource(reaction?.reactionType, reactionResources)
+        : '';
```

Comment:
Can we use undefined here, I assume empty string is not a valid Url so we want to ensure any developer trying to use backgroundImageUrl variable knows that it might be an invalid value
```suggestion
        : undefined;
```